recipes-kernel/linux/linux-at91-5.4.104/sam9x60/defconfig

```diff
@@ -31,9 +31,9 @@ CONFIG_HAVE_KERNEL_LZ4=y
 CONFIG_KERNEL_GZIP=y
 # CONFIG_KERNEL_LZMA is not set
 # CONFIG_KERNEL_XZ is not set
-# CONFIG_KERNEL_LZO is not set
+CONFIG_KERNEL_LZO=y
```

Comment:
Yes, probably to be considered there aswell. I've done a fast test right now and by just removing that (warning: didn't then check it's functionality or anything else) the uImage goes from ~6Mb to ~5.5Mb, so a pretty important chunk!

Reply:
That's very relevant! We can go on removing on V4 too. Anyway, reading the doc it's not very clear to me what means enable or disable this config.

Reply:
Although old I guess this should clarify this: https://bootlin.com/blog/lzo-kernel-compression/

As far as I see LZO, compared for example to gzip, compresses less, but is faster to decompress. But for us probably having a fraction more to have it uncompressed at boot is less important than saving space.

From kernel help:
```
          It's compression ratio is the second poorest amongst the choices. The
          kernel size is about 10% bigger than gzip. Despite that, it's
          decompression speed is the second fastest and it's compression speed
          is quite fast too.
```

Reply:
Yess, Sorry for the dumb question. What i was thinking about Is what happens It gzip and lzo are both enabled. My Guess, looking at the genereted Image size reduction when removing lzo Is that, of both are enabled the. Lzo Is used.

Yess, se should prefer saving bytes, probably the difference in boot Time Is so Low that it's hardly noticeable.

There would be Aldo lzma if we want ti optimize even further, but at the moment i think it's not needed, web can consider It in case we'll be really constrained in kernel size